Fixed demo minor bugs

ValentynaBondarenko opened the pull request:
removed "T" from date ( Order journal)

ValentynaBondarenko committed:
fixed-demo-minor-bugs
1.Delete "T" on date

ValentynaBondarenko committed:
Merge branch 'develop' of github.com:Rooman/PhotoStudio into fixed-demo-minor-bugs

viktorterljuk commented on src/main/java/com/photostudio/web/servlet/order/AllOrdersServlet.java:
```diff
@@ -16,7 +17,7 @@
 public class AllOrdersServlet extends HttpServlet {
-
+    private static final UIutil UI_UTIL = new UIutil();
     private OrderService defaultOrderService = ServiceLocator.getService(OrderService.class);
```
Can we register it in ServiceLocator?

ValentynaBondarenko committed:
fixed-demo-minor-bugs
1.Add dependency "thymeleaf-extras-java8time" fot a data;
2.Add Java8TimeDialect class to TemplateEngine for use #temporals object in our templates;
